### ISSUE TITLE
browser environment settings do not seem to work as expected

### ISSUE THREAD

**Issue by doberkofler** [eslint/eslint#1670]
In my understanding eslint should define console (and more) when setting the browser environment but using the following example files eslint v0.12.0 reports the following unexpected errors:
```
test.js
  1:0  error  'console' is not defined  no-undef
```

.eslintrc
```
{
    "env": {
        "browser": true
    },
    "rules": {
        "no-console": 0
    }
}
```

test.js
```
console.log("hello world");
```

**Comment by nzakas** [comment]
Looks like the globals module is missing `console`. Filed an issue here: https://github.com/sindresorhus/globals/issues/14
